product-service/src/main/resources/application-dev.yml

```
@@ -1,6 +1,50 @@
+  zipkin:
+    tracing:
+      endpoint: http://${ZIPKIN_HOST}:${ZIPKIN_PORT}/api/v2/spans
+  tracing:
+    sampling:
+      probability: 1.0
+  server:
+    port: ${MANAGEMENT_PRODUCT_PORT}
+  endpoints:
+    web:
+      exposure:
+        include: health,prometheus
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)